containerized_ml_models/model_api_8/api.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import FastAPI, Request
+from pickle import load
+import numpy as np
+from uvicorn import run
+
+app = FastAPI()
+model = load(open("model.pkl", "rb"))
```

Comment:
You haven't close the file. Try to use with open(), it will be only 1 line longer and will automatically close the file.